app/ui/browser/reducers/main-reducers.js

```diff
@@ -11,11 +11,12 @@ specific language governing permissions and limitations under the License.
 */
 
 import assert from 'assert';
+import 'babel-polyfill';
```

Comment:
Yucky, yeah -- but let's land it and deal with that later. Sounds like we'd either need a build step or not use these new methods in this case